assembl/static2/js/app/components/debate/survey/question.jsx

```diff
@@ -130,7 +130,7 @@ export class Question extends React.Component<Props, State> {
               <RichTextEditor
                 editorState={this.state.postBody}
                 onChange={this.updateBody}
-                placeHolder={I18n.t('debate.survey.txtAreaPh')}
+                placeholder={I18n.t('debate.survey.txtAreaPh')}
```

Comment:
Good catch. It's silly we have a different case placeHolder for the datepicker. We should lowercase the prop for datepicker field too.